### DISCUSSION TITLE
How to espace curly braces with Icdu format? (`{}`)

### DISCUSSION THREAD

**Discussion by Akronae** [tolgee/tolgee-platform#1552] [Q&A] [answered]
Using `@tolgee/format-icdu` format with React, if the key's value contains any curly braces like
```js
{"hello": 42}
```
A `SyntaxError: MALFORMED_ARGUMENT` error will be thrown, I wonder if this is a bug or just myself not escaping well. Could not find any info about escaping.

**Comment by JanCizmar** [comment, marked as answer]
Hey, you can escape it by using `'` character.

This should do:
```
'{"hello": 42}
```

**Reply by Akronae** [reply]
Works fine, thanks! (adding the missing `'` at the end)
